styleguide/source/assets/scss/01-atoms/_page-title.scss

```diff
@@ -2,4 +2,14 @@
 .ama__page-title {
   @extend %ama__h1;
   @include gutter($padding-top-full...);
+
+  &.allCaps {
```

Comment:
In order to align with [our standards](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/develop/docs/standards.md#how-to-classifyname-patterns) this class should really be `is-all-caps` or `is-uppercase` or something else that would follow BEM.